soccer/gameplay/plays/Defend_Goal/basic_defend_goal.py

```diff
@@ -45,6 +51,9 @@ def __init__(self, num_defenders=3, num_wingers=2):
             self.add_subbehavior(wing_defender.WingDefender(),
                                  'winger' + str(i))
 
+    def use_standard_defense(self):
```

Comment:
Is this supposed to return false?

Reply:
Looking at standard_play.py, no, this function never returns a boolean although you'd think it would based on the title. It possibly should work that way but it does not.